README.md

```diff
@@ -1,2 +1,3 @@
 # best-repo-ever
```

Comment:
These line level comments are a great way to give additional context on recommended changes. They will also be displayed in the conversation view.

Reply:
approved